simulation-scripts/scenario/container-defeat-in-detail/tasks.yaml

```diff
@@ -1,5 +1,9 @@
 objective: Exec into a container on master with certificates mounted in to exploit etcd.
-starting-point: '{"master":[{"as_root": true}]}'
+starting-point:
+  type: outside
```

Comment:
Minor nitpick but I'm not sure 'outside' quite fits, would 'external' be better?